content/en/numpy-code-of-conduct.md

```diff
@@ -0,0 +1,83 @@
+<h2>NumPy Code of Conduct</h2>
+
+<h3>Introduction</h3>
```

Comment:
Use Markdown here, so for a `h3` it's `### Introduction`

Reply:
@rgommers Thank you for all your advice! 

For the Terms of Use page I used termly.io/products/terms-and-conditions-generator/ to fit the needs of numpy.org.

Re: privacy.md My vote goes to the second option - referencing and linking to https://numfocus.org/privacy-policy. I’ll change the text on the Privacy page accordingly.

Reply:
> Use Markdown here, so for a `h3` it's `### Introduction`

Done.